### PR TITLE
feat: ETH stake sent during offer creation and contact details fix

### PR DESCRIPTION
**Pull request by richard-ramos**

### CODE REVIEW

File: src/js/features/metadata/actions.js [hunk 1]
```diff
@@ -26,10 +27,13 @@ export const addOffer = (seller) => ({
     margin: seller.margin,
     arbitrator: seller.arbitrator,
     limitL: seller.useCustomLimits ? seller.limitL.toFixed(2).toString().replace('.', '') : 0,
-    limitU: seller.useCustomLimits ? seller.limitU.toFixed(2).toString().replace('.', '') : 0
+    limitU: seller.useCustomLimits ? seller.limitU.toFixed(2).toString().replace('.', '') : 0,
+    stake: offerStake
```
Comment (jrainville) [inline src/js/features/metadata/actions.js]: I don't think we should pass the stake in the action, the saga should just know what the value is needed

File: src/js/features/metadata/saga.js [hunk 2]
```diff
@@ -236,6 +236,18 @@ export function *onDeleteOffer() {
   yield takeEvery(DELETE_OFFER, doTransaction.bind(null, DELETE_OFFER_PRE_SUCCESS, DELETE_OFFER_SUCCEEDED, DELETE_OFFER_FAILED));
 }
 
+export function *getOfferPrice() {
+  try {
+    const price = yield MetadataStore.methods.getAmountToStake(web3.eth.defaultAccount).call();
```
[the remark this answers is not in the document]
Comment (richard-ramos) [reply src/js/features/metadata/saga.js]: hmm.. I might have forgotten to push that change.

File: src/js/features/metadata/actions.js (hunk 1 above)
Comment (richard-ramos) [reply src/js/features/metadata/actions.js]: good idea!

File: src/js/features/metadata/saga.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (richard-ramos) [reply src/js/features/metadata/saga.js]: https://github.com/status-im/status-teller-network/blob/master/contracts/common/Stakable.sol#L79

**Pull request merged**